### DISCUSSION TITLE
Why does wind up has 0.53 score multiplier?

### DISCUSSION THREAD

**Discussion by mooshiros** [ppy/osu#29469] [Q&A]
Wind up has a 0,53 score multiplier no matter what the starting and ending rates are, which is the same as wind down. I can't tell if this is intentional or a mistake?

**Comment by bdach** [comment]
It's a rudimentary multiplier put in place to prevent people cheesing it if it had 1.0x. Feedback regarding what the final multiplier should be is welcome.

**Reply by felipemarins** [reply]
Is there any correct place to put feedback about final mod multipliers? Is there a existing discussion where it can work as a main place? Do I create a new discussion? Do I put it here? I'm asking because I've seen some people confused about how to do this feedback.

**Reply by bdach** [reply]
Here is fine.

**Comment by felipemarins** [comment]
So I will share my thoughts on the Wind Up/Wind Down mod multiplier. For me, it should use the multiplier for the lowest rate used.
Two examples:

- When a player select the Wind Down mod and set it to 0.8x-0.5x, it should use the same multiplier as the one for Half Time on 0.5x.
- When a player select the Wind Up mod and set it to 1.1x-1.5x, it should use the same multiplier as the one for Double Time on 1.1x.

Someone might also suggest to calculate the average of the rates and use the multiplier for that, but that way it can be somewhat cheesed if, for example, the map has the streak peak at the end and the beginning is much easier than that (which is a somewhat common thing with TV Size maps), so the player would choose Wind Down with 1.8x-1.2x and play a much easier map than with DT 1.5x but still have the same multiplier.

You could also try to make to make weighted average where the lowest rate has the highest weighting factor, but it would make the multiplier too complex and I don't think that's what the dev team wants. In that case, for a safe value, I think that using the multiplier for the lowest rate used is the best solution I could think of.

**Reply by felipemarins** [reply]
Also, while testing the cheesing scenario, I found that the difficulty calculation always ignores the initial rate on Wind Up/Wind Down and just uses the final rate like it is Double Time/Half Time. I don't know if that is a known issue already, but I wanted to report it just in case.
